Uplift/Controllers/GymDetailViewController.swift

```diff
@@ -237,6 +221,63 @@ extension GymDetailViewController {
     }
 }
 
+// MARK: - Item Height Calculations
+extension GymDetailViewController {
+    enum ConstraintConstants {
+        static let dividerHeight = 1
+        static let dividerViewTopPadding = 24
+        static let popularTimesHistogramTopPadding = 24
+        static let noMoreClassesLabelBottomPadding: CGFloat = 57
+    }
+
+    func hoursHeight() -> CGFloat {
+        let baseHeight = CGFloat(GymDetailHoursCell.Constants.hoursTitleLabelTopPadding +
+            GymDetailHoursCell.Constants.hoursTitleLabelHeight +
+            GymDetailHoursCell.Constants.hoursTableViewTopPadding +
+            GymDetailHoursCell.Constants.dividerTopPadding +
+            ConstraintConstants.dividerHeight)
+        let height = gymDetail.hoursDataIsDropped
+            ? baseHeight + CGFloat(GymDetailHoursCell.Constants.hoursTableViewDroppedHeight)
+            : baseHeight + CGFloat(GymDetailHoursCell.Constants.hoursTableViewHeight)
+        return height
+    }
+
+    func busyTimesHeight() -> CGFloat {
+        let labelHeight = GymDetailPopularTimesCell.Constants.popularTimesLabelTopPadding +
+        GymDetailPopularTimesCell.Constants.popularTimesLabelHeight
+
+        let histogramHeight = ConstraintConstants.popularTimesHistogramTopPadding +
+        GymDetailPopularTimesCell.Constants.popularTimesHistogramHeight
+
+        let dividerHeight = ConstraintConstants.dividerViewTopPadding +
+        ConstraintConstants.dividerHeight
+
+        return CGFloat(labelHeight + histogramHeight + dividerHeight)
+    }
+
+    func facilitiesHeight() -> CGFloat {
+        let baseHeight = CGFloat(GymDetailFacilitiesCell.Constants.facilitiesLabelTopPadding +
+            GymDetailFacilitiesCell.Constants.facilitiesLabelHeight +
+            GymDetailFacilitiesCell.Constants.gymFacilitiesTopPadding +
+            ConstraintConstants.dividerViewTopPadding +
+            ConstraintConstants.dividerHeight)
+        let tableViewHeight = GymDetailFacilitiesCell.Constants.gymFacilitiesCellHeight *
+                CGFloat(gymDetail.facilities.count)
+        return baseHeight + tableViewHeight
+    }
+
+    func todaysClassesHeight() -> CGFloat {
+        let baseHeight = CGFloat(GymDetailTodaysClassesCell.Constants.todaysClassesLabelTopPadding +
+            GymDetailTodaysClassesCell.Constants.todaysClassesLabelHeight)
+        return (todaysClasses.isEmpty) ? baseHeight + GymDetailTodaysClassesCell.Constants.noMoreClassesLabelTopPadding +
```

Comment:
Since you have baseHeight representing the sum of a bunch of other constants, why not do the same for baseHeight + ... this stuff ... :     so the ternary looks cleaner.